llvm/lib/Target/PowerPC/PPCSubtarget.cpp

```diff
@@ -194,6 +202,46 @@ bool PPCSubtarget::isGVIndirectSymbol(const GlobalValue *GV) const {
   return true;
 }
 
+CodeModel::Model PPCSubtarget::getCodeModel(const TargetMachine &TM,
```

Comment:
Thanks for addressing the comments. Maybe you are still working on the refactor, but since now you add this function, should we remove the ones in `llvm/lib/Target/PowerPC/PPCAsmPrinter.cpp` and `llvm/lib/Target/PowerPC/PPCISelDAGToDAG.cpp`?